src/plugins/administrative_templates/ui/i18n/filteroptions_en.ts

```diff
@@ -0,0 +1,130 @@
+        <location filename="../filteroptionswidget.ui" line="212"/>
+        <location filename="../filteroptionswidget.ui" line="291"/>
+        <source>No</source>
+        <translation>No</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="179"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the type of policy settings to display.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>Select the type of policy settings to display.</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="220"/>
+        <source>Commented:</source>
+        <translation>Commented:</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="254"/>
+        <source>Enable Keyword Filters</source>
+        <translation>Enable Keyword Filters</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="264"/>
+        <source>Filter for word(s):</source>
+        <translation>Filter for word(s):</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="299"/>
+        <source>Within:</source>
+        <translation>Within:</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="327"/>
+        <source>Policy Settings Title</source>
+        <translation>Policy Settings Title</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="343"/>
+        <source>Help Text</source>
+        <translation>Help Text</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="353"/>
+        <source>Comment</source>
+        <translation>Comment</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="387"/>
+        <source>Enable Requirements Filters</source>
+        <translation>Enable Requirements Filters</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="400"/>
+        <location filename="../filteroptionswidget.ui" line="404"/>
+        <source>Include settings that match any of the selected platforms.</source>
+        <translation>Include settings that match any of the selected platforms.</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="409"/>
+        <source>Include settings that match all of the selected platforms.</source>
+        <translation>Include settings that match all of the selected platforms.</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="423"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the desired platform and application filter(s):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
```

Comment:
It will be cleaner if we disable rich text here.

Reply:
fixed